test/sharness/t0051-object.sh

```diff
@@ -145,6 +161,23 @@ test_object_cmd() {
 		echo QmUNLLsPACCz1vLxQVkXqqLX5R1X345qqfHbsf67hvA3Nn > rmlink_exp &&
 		test_cmp rmlink_exp rmlink_output
 	'
+
+	test_expect_success "multilayer rm-link should work" '
+		ipfs object patch $(cat multi_patch) rm-link a/b/c > multi_link_rm_out
+	'
+
+	test_expect_success "output looks good" '
+		echo "QmZD3r9cZjzU8huNY2JS9TC6n8daDfT8TmE8zBSqG31Wvq" > multi_link_rm_exp &&
+		test_cmp multi_link_rm_out multi_link_rm_exp
+	'
+
+	test_patch_create_path $EMPTY a/b/c $FILE
+	test_patch_create_path $EMPTY a $FILE
+	test_patch_create_path $EMPTY a/b/b/b/b $FILE
```

Comment:
i think we need to have newlines between these. i don't know why, but i've seen errors when I dont have them...